Sources/Sharing/Internal/Reference.swift

```diff
@@ -24,15 +24,16 @@ protocol Reference<Value>:
   #endif
 }
 
-extension Reference {
-  public var description: String {
-    "\(wrappedValue)"
-  }
-}
-
 protocol MutableReference<Value>: Reference, Equatable {
-  var snapshot: Value? { get set }
```

Comment:
The library's internals used an optional getter/setter for change tracking that is better modeled as an optional getter with a non-optional setter.